onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2228,6 +2228,11 @@ static constexpr OrtApi ort_api_1_to_8 = {
     // End of Version 8 - DO NOT MODIFY ABOVE (see above text for more information)
 
     // Version 9 - In development, feel free to add/remove/rearrange here
+    &OrtApis::CreateTensorRTProviderOptions,
+    &OrtApis::UpdateTensorRTProviderOptions,
+    &OrtApis::GetTensorRTProviderOptions,
+    &OrtApis::ReleaseTensorRTProviderOptions,
```

Comment:
The name of the structure is ort_api_1_to_8. Let's make sure that this is handled correctly.

Reply:
Made the name to ort_api_1_to_9, is that correct?

Reply:
Removed these four functions from C API